### PR TITLE
OKTA-76866 Reduce information in the id_token claim if a token claim was requested

### PR DESCRIPTION
**Pull request by mohitchugh-okta**
Clarified which all claims are returned in this case.
Also removed login claim and added preferred_username claim as per some earlier code changes.

Reviewers: @tthompson-okta @joelfranusic-okta 
FYI: @mystiberry-okta

### CODE REVIEW

File: _source/docs/api/resources/oidc.md [hunk 1]
```diff
@@ -119,6 +119,11 @@ This is the digital signature signed by Okta using the public key identified by
 | address     | address    | User's preferred postal address. The value of the address member is a JSON structure containing <em>street_address</em>, <em>locality</em>, <em>region</em>, <em>postal_code</em>, and <em>country</em>.   | JSON structure    | { "street_address": "123 Hollywood Blvd.", "locality": "Los Angeles", "region": "CA", "postal_code": "90210", "country": "US" }     |
 | phone_number     |  phone   | User's preferred telephone number in E.164 format.   | String    | 	"+1 (425) 555-1212"     |
 
+The client can also optionally request an access token along with the id token. This access token allows the client to request claims about the user via a [/oauth2/v1/userinfo](#get-user-information) endpoint. 
+
+When the requested response type contains both the access token and the id token, the id token body does not contain all the scope dependent claims. Instead, only the  <em>name</em> and <em>preferred_username</em> claims if the <em>profile</em> scope was requested and <em>email</em> claim if the <em>email</em> scope was requested are returned.
+
```
Comment (tthompson-okta) [inline _source/docs/api/resources/oidc.md]: We should maybe include a sentence about the rationale for this (i.e. ".. in order to keep the size of the ID token small, you can use the access token against the /userinfo endpoint to fetch the full set of claims", etc).

Comment (mohitchugh-okta) [inline _source/docs/api/resources/oidc.md]: Redid the sentences to include the rationale.

**Comment by tthompson-okta** [comment]
:ship:

**Comment by mohitchugh-okta** [comment]
@mystiberry-okta Yes. That's a mistake, it should have read 'identical to those returned'.

**Pull request merged**